### DISCUSSION TITLE
Where can I find the language pack, I need to translate it into Chinese for use.

### DISCUSSION THREAD

**Discussion by ceozero** [guydavis/machinaris#652] [General]
Where can I find the language pack, I need to translate it into Chinese for use.

machinaris/web/translations/zh/LC_MESSAGES/messages.po   
Modifying this package doesn't seem to work. . I don't know how to compile, how can I quickly translate the language pack to use.

**Comment by guydavis** [comment]
Greetings!  Contributing a localization would be most appreciated.  Please see:

- https://github.com/guydavis/machinaris/wiki/Localization
- https://github.com/guydavis/machinaris/wiki/Contributing#updating-translations

Let me know if you have any questions.  Thanks!
